src/Symfony/Bridge/Doctrine/DependencyInjection/AbstractDoctrineExtension.php

```diff
@@ -352,6 +414,12 @@ protected function loadCacheDriver($cacheName, $objectManagerName, array $cacheD
             case 'xcache':
             case 'wincache':
             case 'zenddata':
+                if ($container->hasParameter($this->getObjectManagerElementName(sprintf('cache.%s.class', $cacheDriver['type'])))) {
+                    @trigger_error(
+                        'Usage of class parameter for '.$cacheDriver['type'].' class is deprecated since version 4.2 and
+                            ill be removed in 5.0. Use configuration values instead.'
```

Comment:
typo: will
please also put all call to trigger_error on a single line to have them consistent with the code base and easy grepping the code

Reply:
OK i'll fix that, any idea what would be the right usage now? I mean i will have to write it in the dreprecation notice.

Reply:
Maybe some inspiration in #11881?